src/components/sessions/activeSession/feedback/FeedbackChannel.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Checkbox, Header } from 'semantic-ui-react'
+
+import Feedback from './Feedback'
+
+import withCSS from '../../../../lib/withCSS'
+
+const FeedbackChannel = ({ data, intl }) => (
+  <div>
+    <Header
```

Comment:
Use h2 etc. directly instead of using the `Header` component. This allows for inline styles.